lib/usd/utils/tests/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+find_package(GTest REQUIRED)
```

Comment:
Tests are living in a separate `test` folder where we match the folder structure from source directory, for example: 
https://github.com/Autodesk/maya-usd/tree/dev/test/lib/usd/utils
https://github.com/Autodesk/maya-usd/tree/dev/test/lib/ufe

Please move all the tests following pattern above.